core/src/main/java/com/microsoft/applicationinsights/TelemetryConfiguration.java

```diff
@@ -126,27 +122,8 @@ public synchronized void setChannel(TelemetryChannel channel) {
         this.channel = channel;
     }
 
-    /**
-     * Gets value indicating whether sending of telemetry to Application Insights is disabled.
-     *
-     * This disable tracking setting value is used by default by all {@link com.microsoft.applicationinsights.TelemetryClient}
-     * instances created in the application.
-     *
-     * @return True if tracking is disabled.
-     */
     public boolean isTrackingDisabled() {
```

Comment:
is this still needed?

Reply:
it's only used for code generation in TelemetryClientClassFileTransformer